assets/js/base/components/checkout/no-shipping/style.scss

```diff
@@ -1,9 +1,19 @@
-.wc-block-checkout__no-shipping .components-placeholder__fieldset {
+.wc-block-checkout__no-shipping {
 	display: block;
```

Comment:
Is `display: block` needed? I think it's always overridden in the last versions of Gutenberg. In old versions, it doesn't play well with the placeholder (screenshot from WP 5.3 with Gutenberg disabled):
![imatge](https://user-images.githubusercontent.com/3616980/77084005-a50e2a00-69fe-11ea-9241-6c37d655cf92.png)